src/app/components/account/monitoring/monitor-chart/monitor-chart.component.ts

```diff
@@ -196,7 +196,8 @@ export class MonitorChartComponent implements OnChanges, OnDestroy {
             },
             tooltips: {
                 enabled: true,
-                mode: "index",
+                mode: "single",
+                position: "nearest",
```

Comment:
Update the tooltip configuration of monitor chart to improve the visibility of the tooltip. Please note that this changes the existing behavior of the tooltip to only show information about the hovered datapoint rather than displaying all datapoints at the hovered index which wasn't helpful because all datapoints could not be displayed within a limited space.

## Before

![Screen Shot 2023-03-23 at 2 19 08 PM](https://user-images.githubusercontent.com/79171711/227365545-ccf4f870-b1c4-4fae-80d4-0f17999e8827.png)

![Screen Shot 2023-03-23 at 2 19 03 PM](https://user-images.githubusercontent.com/79171711/227365554-fb11d267-1b6f-42c7-9419-06192cbb21f6.png)

![Screen Shot 2023-03-23 at 2 18 58 PM](https://user-images.githubusercontent.com/79171711/227365583-8670d190-9339-494d-b9e5-34d3211484c6.png)


## After

![Screen Shot 2023-03-23 at 10 41 16 AM](https://user-images.githubusercontent.com/79171711/227364784-85fabba3-4031-40c7-82b3-fc669a4c9738.png)

![Screen Shot 2023-03-23 at 10 40 50 AM](https://user-images.githubusercontent.com/79171711/227364786-5b69efd5-a87e-4d76-9424-5123cd30fd4c.png)

![Screen Shot 2023-03-23 at 10 41 20 AM](https://user-images.githubusercontent.com/79171711/227364782-b6c1fb6a-bf5e-4c02-9292-e627f0acc1b5.png)